internal/build/buildkit_printer.go

```diff
@@ -37,25 +37,33 @@ type vertex struct {
 }
 
 const internalPrefix = "[internal]"
+const internalLoader = "[internal] load build context"
 const logPrefix = "  → "
 
 var stageNameRegexp = regexp.MustCompile(`^\[.+\]`)
 
 func (v *vertex) isInternal() bool {
```

Comment:
ffti: `isInternal` is a bit inaccurate now - maybe something like `shouldPrint` or `shouldHide`?

Reply:
ahhh good point